hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/scm/protocolPB/StorageContainerLocationProtocolClientSideTranslatorPB.java

```diff
@@ -158,15 +158,6 @@ private ScmContainerLocationResponse submitRpcRequest(
     return response;
   }
 
-  /**
-   * Asks SCM where a container should be allocated. SCM responds with the set
-   * of datanodes that should be used creating this container. Ozone/SCM only
-   * supports replication factor of either 1 or 3.
-   *
-   * @param type   - Replication Type
-   * @param factor - Replication Count
-   */
-  @Override
   public ContainerWithPipeline allocateContainer(
```

Comment:
Just double checking - is this change intentional? I have not looked into the details of this class etc.

Reply:
Thanks. I think it was removed accidentally when I added the new method. Restored it (and also fixed the owner field here...)